.github/workflow-templates/pr-actions.yml

```diff
@@ -10,24 +10,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Run eslint
-        uses: Khan/eslint-action@main
-        setup: yarn
+        uses: Khan/eslint-action@da16c43dd352e1e606c838008e7f25e3bf3fbf97
```

Comment:
Would we revert this to `main` once this lands? or...?

Reply:
yeah I can change it before landing